src/min_circle/welzl.py

```
@@ -47,6 +47,8 @@ def make_circle_n_points(matrix: list[np.ndarray]) -> Circle:
     if num_points == 2 or num_points == 3:
         return min_circle_cvx(np.array(matrix), solver="CLARABEL")
 
+    raise ValueError(f"Expected 0-3 points, got {num_points}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)